pkg/gen/input/workflows/internal/file/create_release.yaml.template

```diff
@@ -158,6 +158,12 @@ jobs:
           file="${{ needs.gather_facts.outputs.project_go_path }}"
           version="${{ needs.gather_facts.outputs.version }}"
           grep -qE "version[[:space:]]*=[[:space:]]*\"$version\"" $file
+      - name: Get Changelog Entry
+        id: changelog_reader
+        uses: mindsers/changelog-reader-action@v2
```

Comment:
When I did this in the past, someone made me aware that `@v2` would allow the "owner" of the action (or an attacker) to replace the code, make a new release, and execute in most of our repositories with our permissions. Which is dangerous.

Instead using a commit SHA is much safer.